Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
```

Comment:
Do you want to continue maintaining this long term?  Does this need to be a person?  Should I find a team email address to put here?

Reply:
copy/paste; but I think it's wiser to put the email 'for this repo' there. As ultimatly, I would love to keep supporting and helping, but it should be 'proper part of' this org, right?

Reply:
Ok I agree with you.  Let's use `clamav-bugs@external.cisco.com`, which is a mailing list we also use for oss-fuzz bug reports (https://github.com/google/oss-fuzz/blob/master/projects/clamav/project.yaml).

Reply:
Perfect! I'll do that.